clang/include/clang/AST/Decl.h

```diff
@@ -3115,7 +3115,19 @@ class FieldDecl : public DeclaratorDecl, public Mergeable<FieldDecl> {
 
   /// Returns the index of this field within its record,
   /// as appropriate for passing to ASTRecordLayout::getFieldOffset.
-  unsigned getFieldIndex() const;
+  unsigned getFieldIndex() const {
+    const FieldDecl *C = getCanonicalDecl();
```

Comment:
Consider keeping the original Canonical variable name.
https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly

Reply:
Done